main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	_ "google-scraper/routers"
+	beego "github.com/beego/beego/v2/server/web"
```

Comment:
How about grouping the import follow our [Go conventions](https://nimblehq.co/compass/development/code-conventions/golang/#imports)?

Reply:
Added extra line between them